src/app/table/table.component.scss

```diff
@@ -378,3 +374,43 @@ a:hover {
   cursor: pointer;
   color: #5C93E8;
 }
+
+.font-size-11 {
```

Comment:
@chandankumar4 I think we need to come up with the naming convention of the class